cmd/lk/app.go

```diff
@@ -268,21 +268,29 @@ func setupTemplate(ctx context.Context, cmd *cli.Command) error {
 
 func cloneTemplate(_ context.Context, cmd *cli.Command, url, appName string) error {
 	var cmdErr error
+
+	tempName, relocate := useTempPath(appName)
+
 	if err := spinner.New().
 		Title("Cloning template from " + url).
 		Action(func() {
-			c := exec.Command("git", "clone", "--depth=1", url, appName)
+			c := exec.Command("git", "clone", "--depth=1", url, tempName)
 			var out []byte
-			if out, cmdErr = c.CombinedOutput(); len(out) > 0 && cmd.Bool("verbose") {
-				fmt.Println(string(out))
+			if out, cmdErr = c.CombinedOutput(); len(out) > 0 {
+				if cmdErr != nil || cmd.Bool("verbose") {
```

Comment:
An alternative could be:
```go
var buf bytes.Buffer
if cmd.Bool("verbose") {
   c.Stdout = os.Stdout
   c.Stderr = os.Stderr
} else {
   c.Stdout = &buf
   c.Stderr = &buf
}
```

Reply:
Unfortunately, we can't use stdout/err directly in this context because the tty has been switched to raw mode and we are in the process of rendering a loading spinner. In the error case its fine because we are about to terminate anyway. But in verbose mode, it scrambles the output.